scripts/tests/integration/imports/test_imports.py

```
@@ -0,0 +1,126 @@
+    """Run a single MKN test manifest.
+
+    Args:
+        name (str): Short name of the test case.
+        manifest (str): Path to the JSON manifest file.
+        expect_fail (bool): True if non-zero exit code is expected.
+
+    Returns:
+        bool: True if the test met expectation, False otherwise.
+    """
+    print(f"test {name} ... ", end="", flush=True)
+    res = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)